src/history/test/HistoryTests.cpp

```diff
@@ -137,6 +137,7 @@ TEST_CASE("History bucket verification", "[history][catchup]")
     VirtualClock clock;
     auto cg = std::make_shared<TmpDirHistoryConfigurator>();
     cg->configure(cfg, true);
+    releaseAssert(cfg.HISTORY_ARCHIVE_RETRY_LIMIT == 10);
```

Comment:
Since this is a default value and we're not testing the value setting, we don't need to assert here. Also, for tests we prefer using `REQUIRE` over `releaseAssert`. It's the same thing, but `REQUIRE` gives you better traces.

I usually like writing a ton of asserts like this during development, but remove them once before I open a PR. It's good to keep `releaseAsserts` to check for things like null pointers, or for situations like if a given function should only be called in a certain scenario. However for sanity checking simple values that won't change, it's a little cleaner to leave these out once everything's working.